docs/sources/configure-server/about-server-api.md

```diff
@@ -213,6 +213,204 @@ curl -X POST \
+| `from`         | UNIX time for the start of the search window                                          | required                                             |
+| `until`        | UNIX time for the end of the search window                                            | optional (default is `now`)                          |
+| `format`       | format of the profiling data                                                          | optional (default is `json`)                         |
+| `maxNodes`     | the maximum number of nodes the resulting flamegraph will contain                     | optional (default is `max_flamegraph_nodes_default`) |
+| `groupBy`      | one or more label names to group the time series by (doesn't apply to the flamegraph) | optional (default is no grouping)                    |
+
+#### `query`
+
+The `query` parameter is the only required search input. It carries the profile type and any labels we want to use to narrow down the output.
+The format for this parameter is similar to that of a PromQL query and can be defined as:
+
+`<profile_type>{<label_name>="<label_value>", <label_name>="<label_value>", ...}`
+
+Here is a specific example:
+
+`process_cpu:cpu:nanoseconds:cpu:nanoseconds{service_name="my_application_name"}`
+
+In a Kubernetes environment, a query could also look like:
+
+`process_cpu:cpu:nanoseconds:cpu:nanoseconds{namespace="dev", container="my_application_name"}`
+
+Note: Take a look at [profiling types]({{< relref "../ingest-and-analyze-profile-data/profiling-types" >}}) for more information.
+
+#### `from` and `until`
+
+The `from` and `until` parameters determine the start and end of the time period that we want to query for.
+They can be provided in absolute and relative form.
+
+**Absolute time**
+
+When passing absolute values we have a few options:
+
+| Option                 | Example               | Notes              |
+|:-----------------------|:----------------------|:-------------------|
+| Date                   | `20231223`            | Format: `YYYYMMDD` |
+| Unix Time seconds      | `1577836800`          |                    | 
+| Unix Time milliseconds | `1577836800000`       |                    | 
+| Unix Time microseconds | `1577836800000000`    |                    | 
+| Unix Time nanoseconds  | `1577836800000000000` |                    |
+
+**Relative time**
+
+Relative values are always expressed as offsets from `now`.
+
+| Option         | Example              |
+|:---------------|:---------------------|
+| 3 hours ago    | `now-3h`             |
+| 30 minutes ago | `now-30m`            |
+| 2 days ago     | `now-2d`             |
+| 1 week ago     | `now-7d` or `now-1w` |
+
+Note that a single offset has to be provided, values such as `now-3h30m` will not work.
+
+**Validation**
+
+The `from` and `until` parameters are subject to these validation rules:
```

Comment:
```suggestion
The `from` and `until` parameters are subject to these validation rules related to `max_query_lookback` and `max_query_length` server properties. You can find more details on these properties in the server docs. 
```

Not sure if we actually mention these parameters in our server docs, but if so we should link to them

Reply:
Good idea! We have them in the "limits" section, will add a link.